pkg/tanka/parallel.go

```diff
@@ -11,44 +12,52 @@ import (
 const defaultParallelism = 8
 
 type parallelOpts struct {
-	JsonnetOpts JsonnetOpts
+	JsonnetOpts
 	Selector    labels.Selector
 	Parallelism int
 }
 
 // parallelLoadEnvironments evaluates multiple environments in parallel
 func parallelLoadEnvironments(paths []string, opts parallelOpts) ([]*v1alpha1.Environment, error) {
+	wg := sync.WaitGroup{}
 	jobsCh := make(chan parallelJob)
-	outCh := make(chan parallelOut)
```

Comment:
For public reference:

A simpler fix to the problem would be to use buffered channels. See https://gobyexample.com/worker-pools for an example

Reply:
I'll have a look.